### PR TITLE
skeletal website for gsu 2023

### PR DESCRIPTION
**Pull request by mjhajharia**
just created the new pages, will insert updated schedule and things soon

### CODE REVIEW

**Comment by mjhajharia** [comment]
@avehtari here's a preview of the updated page https://htmlpreview.github.io/?https://raw.githubusercontent.com/avehtari/BDA_course_Aalto/gsu23schedule/gsu2023.html

**Comment by mjhajharia** [comment]
good thing i saw the github notification, i was about to push a commit which fixes the start/end dates mentioned from last year.

Oh I think one of my commits got over-written because we pushed at the same time!

File: gsu2023.Rmd [hunk 1]
```diff
@@ -0,0 +1,376 @@
...
+All TAs will get a personal certificate from the lecturer Aki Vehtari
+if they actively participate helping students, answering questions,
+and possibly organize some TA sessions. We assume to have enough TAs
+that no-one needs to take part every week of the course and you can
+drop out if other obligations require so. The lecturer and two head
+TAs will support TAs.
+
+To register as volunteer TA fill in your information (email, country,
+prerequisites check, a brief comment on justification why you think
+you can be a TA)  in [TA registration form](https://forms.gle/7KQbnEC3nxb4MwYM9).
```
Comment (harisorgn) [inline gsu2023.Rmd:60]: In the `TA registration form` link, the last question is a "General Comment about you" and it is required. Should we make it optional or was it intentional?

**Review by harisorgn — COMMENTED** [review]
Looks good to me, tried a bunch of links, they all work as expected.

**Review by avehtari — APPROVED** [review]
ok

File: gsu2023.Rmd (hunk 1 above)
Comment (mjhajharia) [reply gsu2023.Rmd:60]: we think it would be useful for context about who the person is, more so than the other things in the form!

**Pull request closed without merging**